dts/msm8916-samsung-r03.dts

```diff
@@ -50,4 +50,15 @@
 			i2c-address = <0x25>;
 		};
 	};
+
+	on7lte-chn {
+		model = "Samsung Galaxy On7 2015 (SM-G6000)";
```

Comment:
Is there also a non-2015 model of On7?

Reply:
[Correct](https://www.gsmarena.com/samsung_galaxy_on7_(2016)-8265.php), it's a rebrand of the J7 Prime with MSM8953 instead of Exynos 7870.